tests/sa.spec.ts

```diff
@@ -8,7 +8,7 @@ const testServiceAccountName = `${testServiceAccountPrefix}${config.sessionID}`;
 
 test.beforeEach(async ({ page }) => {
   await login(page);
-
+  await page.getByRole('link', { name: 'Application and Data Services' }).click();
```

Comment:
Why this is needed?

Reply:
It is a necessary step to get to Kafkas List

Reply:
but this change doesn't seem to be related to change sin this PR so it's some bug fix?

Reply:
I think this change comes from before your PR merge

Reply:
you mean that we introduced the issue in some older PR? Does it cause some test issues?

Reply:
No, I mean that clicking on that button is a necessary step that was added when I was working on this feature even before your code was merged, that's why it shows up here

Reply:
So it seems to me that you did rebase in some wrong way.

Reply:
Because it is part of the function called below this line - https://github.com/redhat-developer/consoledot-e2e/blob/main/lib/sa.ts#L8

Reply:
alright I remove it then

Reply:
Can we resolve this conversation @Frawless ?